manifests/a/Axosoft/GitKraken/8.3.0/Axosoft.GitKraken.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Axosoft.GitKraken
+PackageVersion: 8.3.0
+PackageLocale: en-US
+Publisher: Axosoft, LLC

```

Comment:
```suggestion
Publisher: GitKraken
```
This must match Control Panel.

Reply:
![image](https://user-images.githubusercontent.com/11600822/155741330-054e2b94-b925-4b8e-93b1-a064a3ae8351.png)